deploy/role_binding.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
 subjects:
 - kind: ServiceAccount
   name: cincinnati-operator
-  namespace: openshift-cincinnati
+  namespace: default
```

Comment:
Looks like this was accidentally committed.  I think we want to keep 'openshift-cincinnati'.

Reply:
This was intentional. I wasn't able to deploy the operator as everything else in /deploy uses the default namespace. To use the openshift-cincinnati namespace I think the ServiceAccount and Deployment will also have to switch to openshift-cincinnati.

Reply:
I'm not seeing a namespace designation on any of the objects.  Were you deploying in the default NS?

Reply:
I was using the default. I'll revert back to openshift-cincinnati. We'll need to update the docs so people know they need to deploy the operator to the openshift-cincinnati namespace.